ansible/group_vars/all/h5p-statics.yml

```diff
@@ -0,0 +1,3 @@
+H5P_STATICFILES_SERVER_IMAGE: quay.io/schulcloudverbund/h5p-staticfiles-server
```

Comment:
This line add the same variable to ansible as in ansible/group_vars/all/h5p-staticfiles-server.yml